### DISCUSSION TITLE
ModuleNotFoundError: No module named 'tfel'

### DISCUSSION THREAD

**Discussion by mehranqsb** [thelfer/tfel#387] [General]
Dear Sir,

I installed TFEL/MFront according to what was explained on https://tfel.sourceforge.net/install.html#sec:QuickUbuntu in the 7th section (A Quick installation procedure on Ubuntu). Now, mfm_test_generator_4_1_0 mfront_4_1_0 mtest_4_1_0 std_4_1_0 tfel_4_1_0 are located under ~/.local/lib/python3.10/site-packages. But when I wanted to run pi-plane.py code using the Python command on my UNIX system terminal. I got that "ModuleNotFoundError: No module named 'tfel'". How can I solve this matter?

Kind regards,
Mehran

**Comment by thelfer** [comment]
@mehranghasabeh You specified a "flavour" when compiling TFEL (which appends the version), so you need to be consistent and import tfel_4_1_0

**Reply by mehranqsb** [reply]
Thanks a lot. Now by adding _4_1_0 at the end of tfel, I did not get that message anymore.

Kind Regards,
Mehran

**Reply by mehranqsb** [reply]
Another matter is whether it is possible to eliminate _4_1_0 from the directories. I tried it by moving, for example, tfel_4_1_0 to tfel but as std imported as std_4_1_0 it did not solve it.

Kind regards,
Mehran

**Reply by thelfer** [reply]
Just remove the option `TFEL_APPEND_VERSION` when calling `cmake` :)

**Reply by mehranqsb** [reply]
Thanks I am new to MFRONT so I sometime have an easy question :)

Kind regards,
Mehran

**Reply by thelfer** [reply]
No problem, it's our pleasure to help !

**Reply by mehranqsb** [reply]
Sorry again, but when I ran pi-plane.py code using python3.10, I got this message:
TypeError: No to_python (by-value) converter found for C++ type: tfel::math::vector<double>. Could you help me solve this issue?

I am also new to OpenGeosys and want to run the user material written in mfornt format and use it by the Python-based material subroutine.

Thanks a lot for your help and guidance.
Kind regards,
Mehran

**Reply by thelfer** [reply]
You shall add `import tfel.math`

**Reply by mehranqsb** [reply]
Thanks a lot.

Best Regards,
Mehran
